controllers/Backend.php

```diff
@@ -70,26 +95,42 @@ public function postback()
         $model_article = new Article();
         $model_user = new User();
         
+            if($model_user->admin()!=1){
+        header("Location:/dashboard");
+    }
+
+    $post = $model_article->get_post();
+    if($post == false){
+        header("Location:/page-erreur-administrateur");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The use of function header() is discouraged](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)